src/api/json/catalog.json

```diff
@@ -226,6 +226,16 @@
         "bitbucket-pipelines.yml"
       ]
     },
+    {
+      "name": "bitrise",
+      "description": "The configuration format of the Bitrise CLI. Bitrise is a collection of tools and services to help you with the development and automation of your software projects, with a main focus on mobile apps.",
```

Comment:
I would put CI/CD somewhere here. 🙂

Reply:
This is the official description in our docs, I think this is a deliberate decision to break away from that fixed scope and emphasize our long term goals

Reply:
Makes sense. And WDYT about this description we used here? https://app.bitrise.io/cli